### PR TITLE
sync atbash-cipher

### LINKED ISSUE #839: Sync atbash-cipher
- [x] Run `bin/configlet sync -u -e atbash-cipher --yes --docs --filepaths --metadata --tests include` (updates the Markdown files and maybe `tests.toml`)
  - docs: instructions unsynced: atbash-cipher
- [x] Drop strict types comments from test file and example code (these are useless)
- [x] Add and sync test meta data to tests (`uuid` / `@testdox` in DocBlocks)
- [x] Decide on adding / adjusting / ordering test cases to match current problem specs

Do not redesign the student's interface or add test cases that would invalidate existing community solutions. These are extra tasks, which should be discussed in advance.

### PR DESCRIPTION
**Pull request by tomasnorre**
Resolves #839

### CODE REVIEW

**Comment by tomasnorre** [comment]
For some reason i get the `git rebase` wrong as I wanted to merge in the `ci: Allow for single exercise test exection`. Sorry for that.

File: exercises/practice/atbash-cipher/.meta/example.php [hunk 1]
```diff
@@ -48,3 +26,24 @@ function encode($string)
 
     return implode('', $encoded);
 }
+
+function decode($string)
+{
+    $a_z = range('a', 'z');
+    $z_a = range('z', 'a');
```
Comment (mk-mxp) [inline exercises/practice/atbash-cipher/.meta/example.php:33]: You may use `array_combine()` to make a real lookup table and use `isset()` on it for checking if the input char can be decoded.

**Review by mk-mxp — CHANGES_REQUESTED** [review]
Please add a function stub for `decode()` to the students file.

Everything else looks great, maybe a small improvement to ease the itches of an `array_search()` for each character.

**Comment by tomasnorre** [comment]
Better now?

**Comment by tomasnorre** [comment]
I cannot reproduce the problem with the `composer lint:check` locally.

**Comment by mk-mxp** [comment]
It's about to be solved: [We should follow PHPCS naming conventions as soon as possible](https://github.com/PHPCSStandards/PHP_CodeSniffer/issues/675).

**Comment by tomasnorre** [comment]
> It's about to be solved: [We should follow PHPCS naming conventions as soon as possible](https://github.com/PHPCSStandards/PHP_CodeSniffer/issues/675).
> 
> If you want to provide a PR fixing it really, use `Commenting` category [like here](https://github.com/PHPCSStandards/PHP_CodeSniffer/wiki/Coding-Standard-Tutorial#creating-the-sniff).

I just saw it just now too.

**Comment by tomasnorre** [comment]
> If you want to provide a PR fixing it really, use `Commenting` category [like here](https://github.com/PHPCSStandards/PHP_CodeSniffer/wiki/Coding-Standard-Tutorial#creating-the-sniff).

To be honest, I'm not sure where that fix lives?

**Comment by tomasnorre** [comment]
> If you want to provide a PR fixing it really, use `Commenting` category [like here](https://github.com/PHPCSStandards/PHP_CodeSniffer/wiki/Coding-Standard-Tutorial#creating-the-sniff).

To be honest, i'm not sure how the fix would look like here. 
https://github.com/exercism/php/blob/main/src/Sniffs/ExplainStrictTypesSniff.php

**Comment by tomasnorre** [comment]
This can be review PHPCS is working and MacOS fix is rebased into the PR.

**Review by mk-mxp — APPROVED** [review]
Thanks for all the work!

**Pull request merged**